### DISCUSSION TITLE
src for bin dir

### DISCUSSION THREAD

**Discussion by travelmike123** [Arkady23/V380-Web#29] [General]
Is it possible to post the source to the bin dir? Particularly because I would like to modify the http daemon to perform additional functions. Also, could you specify what developer tools were used to compile the http daemon?

**Comment by Arkady23** [comment]
I took the bin from [this project](https://github.com/TheCrypt0/yi-hack-v4). You can find information about compilation in [the project](https://github.com/shadow-1/yi-hack-v3/issues/23).
